static/app/views/performance/browser/resources/resourceSummaryPage/resourceInfo.tsx

```diff
@@ -0,0 +1,83 @@
+import FileSize from 'sentry/components/fileSize';
+import {Tooltip} from 'sentry/components/tooltip';
+import {t, tct} from 'sentry/locale';
+import {formatBytesBase2} from 'sentry/utils';
+import {RateUnits} from 'sentry/utils/discover/fields';
+import getDynamicText from 'sentry/utils/getDynamicText';
+import {DurationCell} from 'sentry/views/starfish/components/tableCells/durationCell';
+import {ThroughputCell} from 'sentry/views/starfish/components/tableCells/throughputCell';
+import {DataTitles, getThroughputTitle} from 'sentry/views/starfish/views/spans/types';
+import {Block, BlockContainer} from 'sentry/views/starfish/views/spanSummaryPage/block';
+
+type Props = {
+  avgContentLength: number;
+  avgDecodedContentLength: number;
+  avgDuration: number;
+  avgTransferSize: number;
+  throughput: number;
+};
+
+function ResourceInfo(props: Props) {
+  const {
+    avgContentLength,
+    avgDecodedContentLength,
+    avgDuration,
+    avgTransferSize,
+    throughput,
+  } = props;
+
+  const tooltips = {
+    avgContentLength: tct(
+      'On average, this resource is [bytes] when encoded (for example when gzipped).',
+      {
+        bytes: getDynamicText({
+          value: formatBytesBase2(avgContentLength),
+          fixed: 'xx KB',
+        }),
+      }
+    ),
+    avgDecodedContentLength: tct('On average, this resource is [bytes] when decoded.', {
```

Comment:
l: maybe we think about explicitly saying resource body, but I'm fine with this also.

Reply:
Yeah i'm not sure what's best here, to me it's implied the resource content is the body

Reply:
I'll keep it as is for now